common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml

```diff
@@ -202,10 +202,12 @@ eActionOp_APMANAGER:
   _type: enum
   _enum_storage: uint8_t
   ACTION_APMANAGER_INIT_DONE_NOTIFICATION: 0
-  ACTION_APMANAGER_JOINED_NOTIFICATION: 1
-  ACTION_APMANAGER_ENABLE_APS_REQUEST: 2
-  ACTION_APMANAGER_ENABLE_APS_RESPONSE: 3
-  ACTION_APMANAGER_ACK: 4
+  ACTION_APMANAGER_UP_NOTIFICATION: 1
+  ACTION_APMANAGER_CONFIGURE: 2
```

Comment:
This one definitely needs a better name, I just don't know which one since I have no clue what configure means in this regard.

Reply:
It configures the following:
  iface_type
  channel
  iface_filter_low

These are all needed to configure the ap_hal.
The name seems ok to me. One can always look at which data is being sent on this message on the class implementation.

Reply:
Edit: I removed `iface_type` and `iface_filter_low` after @arnout comment.
https://github.com/prplfoundation/prplMesh/pull/1271#discussion_r426739414